src/parser.ts

```diff
@@ -336,5 +352,26 @@ function parseMetaData(headers: string[], footers: string[]): FlightLogMetaData
     camera: {
       serialNumber: findMatch(meta, META_REGEX.cameraSerialNumber),
     },
+    user: {
+      userId: findMatch(meta, META_REGEX.userId),
+      organizationId: findMatch(meta, META_REGEX.organizationId),
+    },
   };
 }
+
+/**
+ * Parse out the given string and return an object if the string is JSON, return undefined otherwise.
+ *
+ * @param info
+ */
+function parseJsonInfo(info: string): object|undefined {
+  if (!info) {
+    return undefined;
+  }
+  try {
+    return JSON.parse(info);
+  }
+  catch (e) {
+    return undefined;
```

Comment:
Can we return the raw value of `info` in this case? Old format flight log files will still be sending non-JSON `info`.

Reply:
No. The return is the *parsed* version of the Info column. I'm using `undefined` as a sentinel to show the parsing failed.